server/routes/api.js

```diff
@@ -2,11 +2,15 @@ const express = require('express');
 const router = express.Router();
 const contacts = require('./contacts');
 const User = require('../database/models/User');
+const passport = require('passport');
+const bcrypt = require('bcryptjs');
+
+const saltRounds = 12;
 
 router.route('/profile')
```

Comment:
You'll probably want to break this out into a `users.js` file once you get to the point where you need to edit the user's information.

Reply:
But for now this is fine.